content/en/monitors/monitor_types/forecasts.md

```diff
@@ -9,39 +9,42 @@ further_reading:
   text: "Configure your monitor notifications"
 - link: "monitors/downtimes"
   tag: "Documentation"
-  text: "Schedule a downtime to mute a monitor"
+  text: "Schedule downtime to mute a monitor"
 - link: "monitors/monitor_status"
   tag: "Documentation"
   text: "Consult your monitor status"
 ---
 
-Forecasting is an algorithmic feature that allows you to predict where
-a metric is heading in the future. It is well-suited for metrics with
-strong trends or recurring patterns.
+Forecasting is an algorithmic feature that allows you to predict where a metric is heading in the future. It is well-suited for metrics with strong trends or recurring patterns.
 
 For example, if your application starts logging at a faster rate, forecasts can alert you a week before your disk fills up, giving you adequate time to update your log rotation policy. Or, you can forecast business metrics, such as user sign-ups, to track progress against your quarterly targets.
 
+To use forecast monitoring, start by creating a [forecast timeseries graph](#forecast-dashboard), and then [set up a monitor](#forecast-monitoring) to trigger when a metric is forecasted to hit a specific threshold.
+
 ## Forecast function
 
 There is a `forecast` function in the Datadog query language. When you apply this function to a series, it returns the usual results along with a forecast of future values.
 
-## Forecast monitor
+## Forecast dashboard
```

Comment:
Could we maybe drop this heading completely, and then talk about the forecast function in both dashboards and notebooks below?

Reply:
Okay, I rearranged and rephrased it based on this feedback - can you please take another look?